### ISSUE TITLE
New Update made the MOD unplayable

### ISSUE THREAD

**Issue by banderitax5** [Aldrin-John-Olaer-Manalansan/DOTA-2-MOD-Master#96]
After this recent update, you can't find a match even an AI match. I'm having the error "Unable to queue for the match when running DOTA 2 as Administrator"

I tried verifying integrity files so the MOD is gone and it's fixed.

@Aldrin-John-Olaer-Manalansan

**Comment by Avtera** [comment]
check my fork

**Comment by banderitax5** [comment]
> check my fork

how does that work

**Comment by Avtera** [comment]
read the instruction

**Comment by gdjn22** [comment]
> check my fork

Hello, after starting the match, how to solve the display VAC

**Comment by DaFiestyBanana** [comment]
> check my fork

@Avtera 
This has since broken since the crownfall update. I assume the modded gameinfo_branchspecific.gi file needs updating everytime the game gets an update?

**Comment by Avtera** [comment]
no it's not, you just need to make sure the gameinfo.gi is not patched by the AJOM mod.
here's how I do it:
- verify the integrity of files
- make the gameinfo.gi file read-only (right click > properties > (in the general tab) tick the read-only option > click apply, done.)

**Comment by gdjn22** [comment]
> 不，不是，您只需要确保 AJOM mod 未修补 gameinfo.gi。我是这样做的：
> 
> * 验证文件的完整性
> * 使 gameinfo.gi 文件为只读（右键单击“属性”>（在“常规”选项卡中）>选中只读选项>单击“应用”、“完成”。

是

**Comment by gdjn22** [comment]
> 不，不是，您只需要确保 AJOM mod 未修补 gameinfo.gi。我是这样做的：
> 
> * 验证文件的完整性
> * 使 gameinfo.gi 文件为只读（右键单击“属性”>（在“常规”选项卡中）>选中只读选项>单击“应用”、“完成”。
That's right gameinfo.gi But there's a lot of stuff in the dota2baet file, can you tell me more about whether it's in that location or if you want to make it all read-only

**Comment by DaFiestyBanana** [comment]
> no it's not, you just need to make sure the gameinfo.gi is not patched by the AJOM mod. here's how I do it:

Ok, interesting, I remember setting the gameinfo.gi file read-only multiple times but this time it worked. Perhaps the game auto-verified after the game crashed the first time when I launched the game after the update and queued it up for when the game was *inactive* on steam (aka I wasn't asking them to verify nor had the game open). 

However, it worked this time. Atleast I know what to do in future, thank you

**Comment by KohakuBeast** [comment]
> However, it worked this time. Atleast I know what to do in future, thank you

are you still playing with this patcher? idk why it doesn't work even with read only

**Comment by DaFiestyBanana** [comment]
Yeah I played game with it fine yesterday. I simply just ran back through the instructions again and noticed that for some reason the file wasn't read-only even though i very much remember setting it so.

All i can recommend is you do the same, double-check it is currently read-only and everything and run back through the instructions.

Sorry I can't help further.

**Comment by KohakuBeast** [comment]
so i just do like this
verify the integrity of files - change gameinfo.gi to read only - open ajom mod - inject skin(not patch gameinfo.gi from ajom) - open fork from avtera - patch the gameinfo.gi - open dota 2 - try playing the game - still have vac warning
so what i do wrong here

**Comment by DaFiestyBanana** [comment]
> open fork from avtera

You need to do this before using ajom mod. And after you do it all, before booting the game, double-check gameinfo.gi is still read-only

**Comment by KohakuBeast** [comment]
> You need to do this before using ajom mod. And after you do it all, before booting the game, double-check gameinfo.gi is still read-only

i got error when i check read-only while patching gameinfo.gi do i need recheck read-only first?

**Comment by DaFiestyBanana** [comment]
> > You need to do this before using ajom mod. And after you do it all, before booting the game, double-check gameinfo.gi is still read-only
> 
> i got error when i check read-only while patching gameinfo.gi do i need recheck read-only first?

Just check after every other step that it is still read-only because the VAC warning can't show up while it is read-only

**Comment by KohakuBeast** [comment]
> Just check after every other step that it is still read-only because the VAC warning can't show up while it is read-only

its still showing vac warning even i check read only idk whats wrong here but can you make video how can you do it, its ok if you don't want to

**Comment by DaFiestyBanana** [comment]
> its still showing vac warning even i check read only idk whats wrong here but can you make video how can you do it, its ok if you don't want to

Did you paste in the file from Avtera's fork? The "gameinfo_branchspecific.gi" file

**Comment by KohakuBeast** [comment]
> Did you paste in the file from Avtera's fork? The "gameinfo_branchspecific.gi" file

i dont see "gameinfo_branchspecific.gi" in avtera fork, alright i see what you mean you mean [this](https://github.com/Avtera/DOTA-2-MOD-Master)

**Comment by DaFiestyBanana** [comment]
![Screenshot_20240502_174045_org mozilla firefox_edit_48396589576468](https://github.com/Aldrin-John-Olaer-Manalansan/DOTA-2-MOD-Master/assets/108105543/fd941084-af0d-4822-ba5c-a8ba764d0dfb)

Check the releases section of his fork

https://github.com/Avtera/DOTA-2-MOD-Master/releases/tag/VAC-Fix

**Comment by gdjn22** [comment]
> 检查我的分叉

After the brother update started the game there is a vac and there is no new patch

**Comment by gdjn22** [comment]
> check my fork

Started vacing again Is there a workaround

**Comment by gdjn22** [comment]
> check my fork

Now matching triggers the vac Is there a workaround

**Comment by operasenpai1** [comment]
no longer working

**Comment by Ozzie25** [comment]
There are several of us who use this program, please let's get it working again. IDK
 what to do anymore. I've tried everything.
